packages/autocomplete-types/index.d.ts

```diff
@@ -7,8 +7,9 @@ declare namespace Fig {
    * - filepaths: show folders and filepaths. Allow autoexecute on filepaths
    * - folders: show folders only. Allow autoexecute on folders
    * - history: show suggestions for all items in history matching this pattern
+   * - help: show suggestions for all the subcommands siblings to the parent one
```

Comment:
I need a feedback on this description, I don't know how to describe it without being too verbose

Reply:
```suggestion
   * - help: show subcommands. Only includes the 'siblings' of the nearest 'parent' subcommand.    
```